environment-dev.yml

```diff
@@ -0,0 +1,10 @@
+# conda env create -f environment-dev.yml
+
+name: PAVICS-landing
+channels:
+  - conda-forge
+  - defaults
+
+dependencies:
+  # then `pre-commit install` to initialize
+  - pre-commit
```

Comment:
In conda-forge, this package is `pre_commit`. Sorry!

Reply:
Oh that's odd, it worked for me.  I actually used that file to create my conda env.

Reply:
That is odd. Well, alright then, haha